src/apps/favorites-list/favorites-list.test.ts

```diff
@@ -45,31 +51,75 @@ describe("Favorites list", () => {
     cy.visit("/iframe.html?id=apps-favorite-list--primary&viewMode=story");
   });
 
+  // Test that the list of favorites is displayed with materials correctly
   it("Favorites list basics", () => {
     // Wait for element not in skeleton screen to prevent testing prematurely.
     cy.get(".cover").should("be.visible");
 
-    // 2.a. Header "Favorites"
+    // Header has "Favorites"
```

Comment:
So this would be something like `it("Shows the correct header")` 
Then we also avoid having comments in the code and these definitions will be printed out in Cypress.